eventservice/api/controllers/events.py

```diff
@@ -290,8 +290,38 @@ def put(event_id):
         __logger.exception(ex)
         abort(400)
 
+    group_memberships = list()
```

Comment:
@bingzhang, I have a couple of comments:
1. At present Group authorization needs to be applied only to group events. I.e., those events with `createdByGroupId` not empty. Non-group events should be updatable like before for now. This might change in the future, but other clients like the Events Manager need support for non-group events for now.
2. Lines 317 and 318 can be combined into a single IF statement, right?

Reply:
what do you mean about 1?

Reply:
When I try to update a non-group event, there is an unauthorized error. A user who is part of the regular AD groups should be able to continue to update events like before. We need to add these group-related authorization only to group events.

Reply:
Updated code based on discussion.